src/server/src/projects/entities/project.entity.ts

```diff
@@ -43,4 +44,11 @@ export class Project implements IProject {
     default: "'{}'"
   })
   lockedDistricts: readonly boolean[];
+
+  // Strips out data that we don't want to have available in the read-only view in the UI
+  getReadOnlyView(): Project {
+    const mutableRoProject = clone(this);
+    mutableRoProject.lockedDistricts = [];
+    return mutableRoProject;
```

Comment:
Using the `...` spread operator can eliminate the use of `lodash` here. Also, consider returning a `lockedDistricts` array w/ a valid number of (false) values.
```suggestion
    return { ...this, lockedDistricts: new Array(this.numberOfDistricts).fill(false) }
```

Reply:
That is nicer, thanks!